ThreatModels/Search.json

```diff
@@ -1 +1,590 @@
+              },
+              "target": {
+                "id": "acc85af3-217e-43e0-a20f-5142b9642096"
+              },
+              "vertices": [
+                {
+                  "x": 203,
+                  "y": 170
+                }
+              ],
+              "id": "017ef625-b484-4507-8c2b-fed28014df79",
+              "labels": [
+                {
+                  "position": 0.5,
+                  "attrs": {
+                    "text": {
+                      "text": "Retreive Request",
+                      "font-weight": "400",
+                      "font-size": "small"
+                    }
+                  }
+                }
+              ],
+              "z": 14,
+              "hasOpenThreats": false,
+              "attrs": {
+                ".marker-target": {
+                  "class": "marker-target hasNoOpenThreats isInScope"
+                },
+                ".connection": {
+                  "class": "connection hasNoOpenThreats isInScope"
+                }
+              }
+            },
+            {
+              "type": "tm.Flow",
+              "smooth": true,
+              "source": {
+                "id": "acc85af3-217e-43e0-a20f-5142b9642096"
+              },
+              "target": {
+                "id": "24f902b3-6eab-4b9b-b502-a919c4fcdc54"
+              },
+              "vertices": [
+                {
+                  "x": 279,
+                  "y": 155
+                }
+              ],
+              "id": "18c25724-28e1-4487-a5e4-1d5ec9759832",
+              "labels": [
+                {
+                  "position": 0.5,
+                  "attrs": {
+                    "text": {
+                      "text": "IRM Response",
+                      "font-weight": "400",
+                      "font-size": "small"
+                    }
+                  }
+                }
+              ],
+              "z": 15,
+              "hasOpenThreats": false,
+              "attrs": {
+                ".marker-target": {
+                  "class": "marker-target hasNoOpenThreats isInScope"
+                },
+                ".connection": {
+                  "class": "connection hasNoOpenThreats isInScope"
+                }
+              }
+            },
+            {
+              "type": "tm.Flow",
+              "smooth": true,
+              "source": {
+                "id": "acc85af3-217e-43e0-a20f-5142b9642096"
+              },
+              "target": {
+                "id": "24f902b3-6eab-4b9b-b502-a919c4fcdc54"
+              },
+              "vertices": [
+                {
+                  "x": 359,
+                  "y": 165
+                }
+              ],
+              "id": "3b2a0fd7-d7cc-4443-8a10-9912a6fa934a",
+              "labels": [
+                {
+                  "position": 0.5,
+                  "attrs": {
+                    "text": {
+                      "text": "Fetch File",
```

Comment:
Sounds like we will have a group review of threat models. I'll use GitHub to jot down questions I'd like to ask when that happens.
What is the "Fetch file" message?

Reply:
It is a GET to the StoreController with the product ID... This isn't implemented yet as far as I can tell, but @emmberk said it needed to be included.

Reply:
Actually that would be the `Retrieve Request` portion, but the fetch file message would be a 200 Response with the content's media type in the header and the retrieve response stream in the body.